<launch>
  <node pkg="cv_camera" name="camPi_USB_cam1" type="cv_camera_node" output="screen">
    <param name="rate" value="30" />
    <param name="device_id" value="0" />
    <!-- <param name="device_id" value="1" /> -->
    <param name="frame_id" value="camera_primer" />
    <param name="image_width" value="320" />
    <param name="image_height" value="240" />
    <!-- <param name="camera_info" value="???" /> -->
    <!-- <param name="file" value="???" /> -->
  </node>

  <node pkg="cv_camera" name="camPi_USB_cam2" type="cv_camera_node" output="screen">
    <param name="rate" value="30" />
    <param name="device_id" value="1" />
    <!-- <param name="device_id" value="1" /> -->
    <param name="frame_id" value="camera_primer" />
    <param name="image_width" value="320" />
    <param name="image_height" value="240" />
    <!-- <param name="camera_info" value="???" /> -->
    <!-- <param name="file" value="???" /> -->
  </node>

  <node name="dynamic_reconfigure" pkg="dynamic_reconfigure" type="reconfigure_gui" />

</launch>
